src/status_im/data_store/realm/schemas/account/migrations.cljs

```diff
@@ -240,3 +176,97 @@
                                                     "status = \"received\""))
                                     (.-length))]
         (aset chat "unviewed-messages-count" user-statuses-count)))))
+
```

Comment:
I take this is a copy & paste of the code from somewhere else, if that's the case, could you add a comment begin/end of copy and paste code?

Reply:
some comments are added, let me know if it's not sufficient enough